src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -2532,423 +2527,475 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
         }
     }
 
+    // Set up and process ZoneMixing and ZoneCrossMixing inputs
+
     RepVarSet = true;
 
     cCurrentModuleObject = "ZoneMixing";
-    state.dataHeatBal->TotMixing = state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, cCurrentModuleObject);
-    state.dataHeatBal->Mixing.allocate(state.dataHeatBal->TotMixing);
-
-    for (int Loop = 1; Loop <= state.dataHeatBal->TotMixing; ++Loop) {
-
-        state.dataInputProcessing->inputProcessor->getObjectItem(state,
-                                                                 cCurrentModuleObject,
-                                                                 Loop,
-                                                                 cAlphaArgs,
-                                                                 NumAlpha,
-                                                                 rNumericArgs,
-                                                                 NumNumber,
-                                                                 IOStat,
-                                                                 lNumericFieldBlanks,
-                                                                 lAlphaFieldBlanks,
-                                                                 cAlphaFieldNames,
-                                                                 cNumericFieldNames);
-        UtilityRoutines::IsNameEmpty(state, cAlphaArgs(1), cCurrentModuleObject, ErrorsFound);
-
-        state.dataHeatBal->Mixing(Loop).Name = cAlphaArgs(1);
+    int numZoneMixingObjects = 0;
+    EPVector<InternalHeatGains::GlobalInternalGainMiscObject> zoneMixingObjects;
```

Comment:
This is local (i.e., not `state`) data?

Reply:
Correct. These are temporary input object stubs used to expand the object for multiple spaces due to Zone or ZoneList or SpaceList name.